OpenChange/MAPIStoreContext.m

```diff
@@ -114,7 +113,6 @@ + (struct mapistore_contexts_list *) listAllContextsForUser: (NSString *)  userN
 
   userContext = [MAPIStoreUserContext userContextWithUsername: userName
                                                andTDBIndexing: indexing];
-  [userContext activateWithUser: [userContext sogoUser]];
```

Comment:
This change makes the `userContext` getter useless according to the compiler. See:
```
 Compiling file MAPIStoreContext.m ...
MAPIStoreContext.m: In function ‘+[MAPIStoreContext listAllContextsForUser:withIndexing:inMemCtx:]’:
MAPIStoreContext.m:110:25: warning: variable ‘userContext’ set but not used [-Wunused-but-set-variable]
   MAPIStoreUserContext *userContext; 
```